app/models/pageflow/published_entry.rb

```diff
@@ -25,6 +25,7 @@ class PublishedEntry
              :author, :publisher, :keywords,
              :theme,
              :password_protected?,
+             :created_at, :updated_at,
```

Comment:
Perhaps delegate `created_at` to Entry instead? 🤔

Reply:
Either way, we should make sure that `PublishedEntry#updated_at` changed when the underlying `Entry` changes (i.e. a feature flag is enabled or the title is changed). The way it is now, only `Revision` changes are accounted for.